README.md

```diff
@@ -197,9 +213,40 @@ output instead:
 ,,,1554210895
 ```
 
-It is possible to control the number of records to be put in a
-particular output file by setting `file.max.records`. By default, it is
-`0`, which is interpreted as "unlimited".
+**NB!**
+
+ - The `key.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter`
+or `org.apache.kafka.connect.storage.StringConverter` for this data format.
+
+ - The `value.converter` property must be set to `org.apache.kafka.connect.converters.ByteArrayConverter` for this data format.
+
+#### **JSONL** Format example
```

Comment:
Nit: bold selection won't be visible in the rendered text.

Reply:
- [ ] Remove bold